lib/hawkular/alerts/alerts_api.rb

```diff
@@ -246,8 +246,18 @@ def get_alerts_for_trigger(trigger_id) # TODO: add additional filters
     # @param [Hash]criteria optional query criteria
     # @return [Array<Alert>] List of alerts in the system. Can be empty
     def list_alerts(criteria = {})
```

Comment:
Not really, `list_alerts` is being called with a hash. In `alerts` method I used keyword arguments. In methods with keyword arguments, ruby tries to match hashes to keyword arguments. If you do the alias, `alerts_client.list_alerts('thin' => true)` (which is in the specs) will be equivalent to `alerts_client.alerts('thin' => true)` and ruby will complain with an exception saying that there is no 'thin' argument. You are forced to specify:
`alerts_client.alerts(criteria: {'thin' => true})`
So, signatures are not very compatible.